std/process.d

```diff
@@ -338,7 +339,131 @@ version(Posix) private enum InternalError : ubyte
     noerror,
     exec,
     chdir,
-    getrlimit
+    getrlimit,
+    doubleFork
+}
+
+version(Posix)
+{
+private:
+    const(char)[] getExecutablePath(const(char)[] name)
```

Comment:
Since there is no longer an overload set, these refactorings aren't needed any more, right? If so, can they be undone?